Dockerfile

```diff
@@ -42,6 +42,7 @@ COPY ibu-imager/cmd ibu-imager/cmd
 COPY ibu-imager/ops ibu-imager/ops
 COPY ibu-imager/common ibu-imager/common
 COPY ibu-imager/seedcreator ibu-imager/seedcreator
+COPY ibu-imager/seedcleanup ibu-imager/seedcleanup
```

Comment:
just a nit, but... Can we just do ```COPY ibu-imager ibu-imager``` here? This is just the builder, so it wouldn't end up with additional content in the final image